### ISSUE TITLE
Wrapped VST3s not showing up in Ableton Live

### ISSUE THREAD

**Issue by tas-from-planet-u-he** [free-audio/clap-wrapper#30]
Tested on macOS Monterey 12.6.1 with a 2020 M1 Silicon MacBook Pro using Ableton Live 11.2.

Wrapped VST3 plugins are being scanned, but then do not show up in Live's VST3 plugin list.
If you replace them with the real VST3 versions, they show up.
Put the wrapped version back in place, and they will vanish from the plugin list again.
Tested with multiple u-he Plugins, same for all of them.

The same wrapped VST3s show up and load without problems in Bitwig 4.4, Reaper and Cubase 12.

**Comment by defiantnerd** [comment]
I've tried to reproduce this with the plugin you've mentioned, and it looks like it is a simple naming issue. The wrapper binary and the CLAP binary do really need the exact same filename stem.

I will add a feature to display a real error when the underlaying CLAP is not being found at all, so it gets more obvious instead of just returning a nullptr.

Additionally, Ableton Live seems not to follow the symlinks the VST3 SDK does create. All other hosts do that.

To test this, I removed the symlinks and copied the actual binary into the VST3 folder and Ableton Live then scans and opens the plugins.

So, this is more an issue with Ableton Live and the way the SDK works. Nevertheless, I will add some better ways to debug this.
